prime-router/settings/staging/0170-ucsd-onboarding-receiver-settings.yml

```diff
@@ -0,0 +1,101 @@
+# To submit this to staging, run:
+# ./prime multiple-settings set --env staging --input ./settings/staging/0170-ucsd-onboarding-receiver-settings.yml
+#
+# To submit this in localhost, run:
+# ./prime multiple-settings set --input ./settings/staging/0170-ucsd-onboarding-receiver-settings.yml
+#
+# Description:
+#   These following settings are for UCSD
+#   Result Receiver (ucsd.etor-nbs-results)
+#
+---
+- name: "ucsd"
+  description: "University of California San Diego"
+  jurisdiction: "STATE"
+  stateCode: CA
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders: []
+  receivers:
+    - name: "etor-nbs-results"
+      organizationName: "ucsd"
+      topic: "etor-ti"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "metadata/hl7_mapping/ORU_R01/ORU_R01-base"
+        useTestProcessingMode: false
+        useBatchHeaders: false
+        receivingApplicationName: "EPIC-INNERCONNECT"
+        receivingApplicationOID: null
+        receivingFacilityName: "CA"
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: {}
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "standard"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01')"
+        - "(%performerState.exists() and %performerState = 'CA') or (%patientState.exists() and %patientState = 'CA')"
+        - "Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(value = 'ucsd-receiver-id').exists()"
```

Comment:
Instead of `ucsd-receiver-id`, this should be the universal id (OID) for UCSD. Also, you don't need the other filter: `(%performerState.exists() and %performerState = 'CA') or (%patientState.exists() and %patientState = 'CA')`

Reply:
Yes. I will confirm their id. This will change once we push it to staging. For now, this is just our local environment.

Reply:
Unfortunately CDPH and UCSD isn't using OIDs. More on that in the transformation spreadsheet linked on [#1024](https://github.com/CDCgov/trusted-intermediary/issues/1024).

Also, we shouldn't use patientState for ETOR. Routing needs to be based on the healthcare provider's facility, not the patient's residence. (e.g. a baby born in Alabama needs screening sent to Alabama's PHL, even if they live out of state)